qarysite/qary_app/migrations/0001_initial.py

```diff
@@ -11,6 +13,15 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
+        migrations.CreateModel(
+            name='Chat',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('question', models.TextField()),
+                ('answer', models.TextField()),
+                ('create_date', models.DateTimeField(default=datetime.datetime(2020, 4, 24, 20, 25, 2, 147005, tzinfo=utc))),
+            ],
+        ),
         migrations.CreateModel(
             name='Post',
             fields=[
```

Comment:
Don't forget to run migrations after each new one you create. `python manage.py migrate`